docs/project/plugins.sbt

```diff
@@ -0,0 +1,5 @@
+/*
+ * Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+addSbtPlugin("com.typesafe.play" % "play-docs-sbt-plugin" % sys.props.getOrElse("play.version", "2.6.0-SNAPSHOT"))
```

Comment:
Actually this is the correct version. There are two sbt builds in this project now, one in the root directory and one in the `docs` directory.

1. The one in the root directory runs without a dependency on Play. It does run the doc tests ([see comment](https://github.com/playframework/play-json/pull/37/files#r103129895)). This is the build file used to test and compile Play JSON.

2. The one in the `docs` directory isn't run by Travis, it's only used when you're developing so you can preview the docs. It has a dependency on Play framework's sbt doc plugin so there's a cycle… In order to break the cycle, first you publish a snapshot of Play JSON using the root sbt settings, then you publish a snapshot of Play. Only then can you run the build in the `docs` directory. It will use the snapshots of Play and Play JSON. See `docs/README.md` for more info.